runtime-light/tl/tl-types.h

```diff
@@ -275,7 +280,10 @@ struct K2JobWorkerResponse final {
 
 // Actually it's "Maybe (Dictionary CertInfoItem)"
 // But I now want to have this logic separately
-struct GetPemCertInfoResponse final {
+class GetPemCertInfoResponse final {
```

Comment:
Why both structs and classes here?

Reply:
Just to not introduce a lot of `public, private` annotations

Reply:
Seems strange a little, but OK